conf/application.conf

```diff
@@ -22,12 +24,15 @@ play.filters.enabled += "play.filters.cors.CORSFilter"
 play.filters {
   csrf.header.bypassHeaders {
     X-Requested-With = "*"
+    Csrf-Token = "nocheck"
   }
   hosts {
-    allowed = ["rcruitme:9000", "localhost:9000"]
+    allowed = ["."]
   }
+  cors.exposedHeaders = ["X-Auth-Token"]
```

Comment:
When I changed removed the proxy in the front end and added the base url, the token wasn't getting exposed. not super sure why.